metadata-ingestion/src/datahub/ingestion/graph/client.py

```diff
@@ -1445,6 +1445,51 @@ def run_assertions_for_asset(
 
         return res["runAssertionsForAsset"]
 
+    def get_entities_v2(
+        self,
+        entity_name: str,
+        urns: List[str],
+        aspects: List[Type[Aspect]],
+        with_system_metadata: bool = True,
+    ) -> Optional[Dict]:
+        payload = {
+            "urns": urns,
+            "aspectNames": list(map(lambda x: x.ASPECT_NAME, aspects)),
```

Comment:
is this required? e.g. can we not specify aspects and just get everything back?

Reply:
Hm, we definitely can do this, but that would be costly. We need to fetch all aspects from the database and transfer over network only to throw them away later. What's concern with this?

Reply:
I've tested the endpoint, it does accept empty list of aspects, and just returns everything back, but like I said that would be very expensive. The use case I'm working on is a great example: one of the aspects of ExecutionRequest is ExecutionRequestResults, which may be many megabytes in size per urn.  If we always fetch, we will have to transfer all these data over network and throw it away, so I'd not drop it here. 

However, we may need to handle case of empty aspect list when building result dict since underlying endpoint supports it.